### DISCUSSION TITLE
Not able to start podman service on macOS 12.6

### DISCUSSION THREAD

**Discussion by dfontenot** [Homebrew/discussions#4068] [Everyday usage] [answered]
Hello,

I am no longer able to run the podman service on my Intel Monterey MBP. I've tried a `brew reinstall podman` but my issue still remains. I have qemu installed via brew as well. Ran a `brew update` and `brew upgrade` too.

If I modify the .plist for the service that brew generates (~/Library/LaunchAgents/homebrew.mxcl.podman.plist), and set the stdout and stderr to a file (side question: why isn't this enabled by default?), and run via launchctl I only see the following on the stderr:

```
Error: unknown flag: --time
See 'podman system --help'
```

Is this a problem with the recipe for podman, or an issue somewhere on my machine? Thanks for reading this.

**Comment by cho-m** [comment, marked as answer]
This has also been reported as a `homebrew-core` issue https://github.com/Homebrew/homebrew-core/issues/118717

At this point, we need someone who uses `podman` on macOS to contribute a PR that they tested works.
